appmgr/src/net/wifi.rs

```diff
@@ -517,6 +517,15 @@ pub fn country_code_parse(code: &str, _matches: &ArgMatches<'_>) -> Result<Count
     )))
 }
 
+pub async fn synchronize_wpa_supplicant_conf(main_datadir: &std::path::Path) -> Result<(), Error> {
+    let source = main_datadir.join("wpa_supplicant.conf");
+    if tokio::fs::metadata(&source).await.is_err() {
+        tokio::fs::write(&source, include_str!("wpa_supplicant.conf.template")).await?;
```

Comment:
does this need to be interpolated? if not, why .template?

Reply:
convention? 🤷🏼‍♂️ I can remove the .template but the point is that its a base config not the full thing.